### PR TITLE
{lib}[foss/2023a] rMATS-long v1.0.0-20240502 w/ R 4.3.2

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/355

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/r/rMATS-long/rMATS-long-1.0.0-foss-2023a-R-4.3.2.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+easyblock = 'Tarball'
+
+name = 'rMATS-long'
+version = '1.0.0'
```
Comment (boegel) [inline easybuild/easyconfigs/r/rMATS-long/rMATS-long-1.0.0-foss-2023a-R-4.3.2.eb]: If we take a more recent commit that the `1.0.0` tag, we should also add a datestamp to the version:
```
version = '1.0.0-20240502'
```

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/rMATS-long/rMATS-long-1.0.0-foss-2023a-R-4.3.2.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/r/rMATS-long/rMATS-long-1.0.0-foss-2023a-R-4.3.2.eb]: fixed in ef85e1f

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3158.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/0c67b124a12c2234df3ea62c06d8de17 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**